README.md

```diff
@@ -1,2 +1,54 @@
-# Hello-world
-testing
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# HELLO MICROVERSE
+
+> setup to test git features .
+
+![image](https://user-images.githubusercontent.com/59481426/148063440-51643a84-4032-490a-b93c-0a90381c8b17.png)
+
+
+
+
+## Built With
+
+- HTML,CSS
+
+- HTML,CSS,GITHUB
+
+## Live Demo
+
+[Live Demo Link](https://lucidkode.github.io/Hello-world/)
+
+
+
+
+
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Lucidkode)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
## [OPTIONAL]

- [ ] Kindly I would suggest you customize this section with your own information in your `README` file